Makefile

```diff
@@ -73,3 +78,7 @@ run-ui-dev:
 # Builds the user interface on your local machine.
 build-ui:
 	cd ui; npm run build && npm run export; cd ..
+
+# setup wrk for local dev
+setup-wrk:
```

Comment:
Is this setup for wrk or wrk2?

Anything needed for building wrk2?

Reply:
I started off with this, but it does not work on a Mac atm. I will get this fixed one way or the other.